content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,459 @@
+---
+reviewers:
+- phanama
+- wahyuoi
+title: Horizontal Pod Autoscaler
+feature:
+  title: Horizontal scaling
+  description: >
+    Scale your application up and down with a simple command, with a UI, or automatically based on CPU usage.
```

Comment:
this needs translation as well because it will show up on homepage
![image](https://user-images.githubusercontent.com/3192347/84157116-87e47700-aa94-11ea-9c30-21ee1526e00f.png)

Reply:
Translated, please check